index.html

```diff
@@ -29,5 +29,106 @@ <h4>let's connect</h4>
+                <ul class="set-stack">
+                    <li class="stack">html</li>
+                    <li class="stack">css</li>
+                    <li class="stack">javascipt</li>
+                </ul>
+                <button class="btn" type="submit">see project</button>
+            </header>
+        </section>
+    
+        <section class="skills">
+            <img class="sample" src="./images/Snapshoot3.png" alt="project snapshoot">
+            <header class="project-header">
+                <h2>tonic</h2>
+                <ul class="tags">
+                    <li class="diff-li">canopy</li>
+                    <li>back end dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="set-stack">
+                    <li class="stack">html</li>
+                    <li class="stack">css</li>
+                    <li class="stack">javascipt</li>
+                </ul>
+                <button class="btn" type="submit">see project</button>
+            </header>
+        </section>
+    
+        <section class="skills">
+            <img class="sample" src="./images/Snapshoot4.png" alt="project snapshoot">
+            <header class="project-header">
+                <h2>tonic</h2>
+                <ul class="tags">
+                    <li class="diff-li">canopy</li>
+                    <li>back end dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="set-stack">
+                    <li class="stack">html</li>
+                    <li class="stack">css</li>
+                    <li class="stack">javascipt</li>
+                </ul>
+                <button class="btn" type="submit">see project</button>
+            </header>
+        </section>
+    </div>
+
+    <section class="about-myself banner">
+        <h2 class="intro">About<br> Myself!
+        </h2>
+        <p class="dev">
+            I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+        </p>
+        <h4>let's connect</h4>
+        <ul>
+            <li><a href="#"><img src="./images/twitter.svg" alt="twitter"></a></li>
+            <li><a href="#"><img src="./images/linkedin.svg" alt="linkedin"></a></li>
+            <li><a href="#"><img src="./images/Vector1.svg" alt=""></a></li>
+            <li><a href="#"><img src="./images/github.svg" alt="github"></a></li>
+            <li><a href="#"><img src="./images/Vector3.svg" alt=""></a></li>
+        </ul>
+        <button class="btn" type="submit">get my resume</button>
+        <ul class="pro-list">
+            <li class="pro-item">languages</li>
```

Comment:
- [ ] Kindly add the languages list in order to stick to the provided Figma design. :wink: 

  - ### Your design
  
   ![image](https://user-images.githubusercontent.com/87197412/185748378-24b5dc97-947e-41cb-94a2-e8de25ef31dc.png)



   - ### Figma design
   ![image](https://user-images.githubusercontent.com/87197412/185748356-5fc12f48-a144-46a7-aee9-e6ca44e4467d.png)

Reply:
@levy002 I have applied all changes but I wish to understand how you want me to put in place these languages and what tags should I use?

Reply:
Hi @Allenkeys, since it is a list of languages, so you should use a `<ul>` tag and inside it `<li>` tags for each language.

Reply:
I used <ul> and the linters flagged it wrong, so I used only <li>

Reply:
@Allenkeys You need to use `ul` tag inside the `li` tag. Something like this

```
<li class="languages">
    <ul>
        <li><img src="./images/javascript.png" alt="javascipt"></li>
        <li><img src="./images/html.png" alt="html"></li>
        <li><img src="./images/css.png" alt="css"></li>
    </ul>
</li>
```